libs/pages/application/src/lib/feature/import-environment-variable-modal-feature/utils/form-check.spec.ts

```diff
@@ -0,0 +1,169 @@
+import { validateKey, warningMessage } from './form-check'
+import { EnvironmentVariableScopeEnum } from 'qovery-typescript-axios'
+
+const mockedExistingEnvVar: any = [
```

Comment:
It's gonna happen a lot of time in test I think that to mock we are gonna just take the API call response and copy paste it here. And sometimes doing that does not match exactly with the type that we use because maybe we extended it a little. I think that in test, any can be accepted but okay to debate. Here typing comes at the cost that I have to manually edit all the entries...